__mocks__/mockCreators/shared.ts

```diff
@@ -57,6 +58,24 @@ export function createIProfile(): imperative.IProfileLoaded {
     };
 }
 
+export function createInvalidIProfile(): imperative.IProfileLoaded {
+    return {
+        name: "sestest",
+        profile: {
+            type : "zosmf",
+            host: null,
+            port: 1443,
```

Comment:
I hope it doesn't cause confusion having different ports for different mockCreators. :yum:
No changes requested

Reply:
I think the first one (443) was just a typo. Fixing anyway :)